UI/window-basic-main.cpp

```diff
@@ -2938,7 +2938,7 @@ void OBSBasic::CloseDialogs()
 		projector.clear();
 	}
 
-	delete stats;
+	if (!stats.isNull()) stats->close(); //call close to save Stats geometry
```

Comment:
I thought that Qt::WA_DeleteOnClose can do it automatically when event accepted...